operator/cmd/syncclusterconfig/sync.go

```diff
@@ -410,3 +434,87 @@ func (s *Syncer) maybeMergeSuperusers(ctx context.Context, config map[string]any
 
 	config[superusersEntry] = normalizeSuperusers(append(superusers, mapConvertibleTo[string](logger, superusersAny)...))
 }
+
+func hashConfig(m map[string]any, schema rpadmin.ConfigSchema) (string, error) {
```

Comment:
Can we give this a comment or more descriptive name? `hashConfig` is quite misleading relative to what it actually does.